students/larry_mburu/Lesson04/mailroom.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+
+import sys
+import tempfile
+import os
+from operator import itemgetter
+
+donations_per_individual = { 
+    'William Gates III' : [50000.0, 500.12], 
+    'Mark Zuckerberg' : [20000.0, 8500.99], 
+    'Jeff Bezo': [100000.0, 40000.0, 700.99], 
+    'Paul Allen': [200000.0, 1440.0, 300.00], 
+    'Bill Gates': [30000.0, 70000.0, 450.0]
+}
+
+def display_menu():
+    """
+    Display menu option to user
+    """
+    print("Welcome to Mail Automator (MA)!")
+    print()
+    print("[1] Send a Thank You!")
+    print("[2] Create a report!")
+    print("[3] Send letters to all donors")
+    print("[4] Quit!")
+    print()
+
+def letter_validator(file_tmp_dir):
+    """
+    Function just prints the content of a single letter in file, for 
+    validation only, since the send_thank_you_letter_to_all() utilizes 
+    tempfile.TempDirectory(), which does not persist files in the file system
+
+    param: file_tmp_dir: temporary directory full path name. 
+
+    return: None
+
+    """
+
+    for file in os.listdir(file_tmp_dir): 
```

Comment:
avoid using `file` for var name since it's a built in variable (notice different color on github)